src/Core/Framework/DataAbstractionLayer/RuntimeEntityExtension.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Shopware\Core\Framework\DataAbstractionLayer;
+
+/**
+ * @internal will be generated when BulkEntityExtension are registered
+ */
+class RuntimeEntityExtension extends EntityExtension
```

Comment:
interesting note:
We probably won't need this class explicitly we could also use an anonymous class inside the registry for that one, that way we don't need to create a whole separate file etc only for this

https://www.php.net/manual/en/language.oop5.anonymous.php

Reply:
fyi: Was also my favorite, didn't expect you would also like it this way ;)